examples/apps/snapshotFetchPerfApp/src/fetchApp.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FluidAppOdspUrlResolver } from "@fluid-tools/fluidapp-odsp-urlresolver";
+import { assert } from "@fluidframework/common-utils";
+import { prefetchLatestSnapshot } from "@fluidframework/odsp-driver";
+import { MockLogger } from "@fluidframework/telemetry-utils";
+import { OdspSampleCache } from "./odspPersistantCache";
+
+export function start(
+    div: HTMLDivElement,
+    odspAccessToken: string,
+) {
+    const binaryDiv = document.createElement("div");
+    binaryDiv.style.minHeight = "400px";
+    const binaryText = document.createElement("div");
+    binaryText.textContent = "Paste Your URL for Binary Snapshot Here";
+    binaryText.style.minHeight = "25px";
+    binaryText.style.marginBottom = "5px";
+    binaryText.style.marginTop = "10px";
```

Comment:
nit: usually these settings live in a separate .css file and we would import them here. But as it is a test app I'd say it is not required